### DISCUSSION TITLE
Creating & Adding additional DISKS to a pre-built image

### DISCUSSION THREAD

**Discussion by MrMicroChip** [prirun/p50em#35] [General]
I'm a former Prime administrator, thoroughly enjoying the nostalgia of running PRIMOS again!  I'm using the Rev 24 pre-built image, and was attempting to create an additional disk using MAKE.  Specifically a MODEL_4475 at Controller '26 Unit 1 (pdev 4463).  But I'm not sure whether the emulator supports that, as I'm just getting MAKE errors.  Can you shed any light as to whether what I'm trying to do is possible?

**Comment by drboone** [comment]
It should work fine.  You just need to `touch disk26u1.315m` before you start the MAKE.

**Reply by MrMicroChip** [reply]
Many thank, Dennis.  I hadn't made the mental connection between the external file, and what it represented within the emulator.  All works fine now.

And a quick 'thank you' to you, Jim, and the others who have contributed so much to bringing all this together.  For those with an interest it really is a gold mine.

Rgds, John Appleyard
